ChangeLog

```diff
@@ -1,52 +1,65 @@
-07/09/2016 - compatibility with Coq 8.7 and several small fixes -
-	version 1.6.2 and upcomming version 1.7
-	* Compatibility with Coq 8.7
+23/04/2018 - compatibility with Coq 8.7 and several small fixes -
+	upcoming version 1.7
+	* Compatibility with Coq 8.8 (and >= 8.6)
 	* Lost compatibility with Coq 8.4
 
-07/09/2016 - compatibility with Coq 8.7 and several small fixes -
-	upcomming version 1.7
-	* New Theorems:
-	   dec_factor_theorem, abstract_context,
-	   mul_bin_down, mul_bin_left
+	* Integration to Coq: ssrbool.v ssrfun.v and plugin.
+	  ssrtest also moved to Coq test suite.
+
+	* Library refactoring:
+	odd_order and real_closed: are now in separate repositories.
```

Comment:
Since real_closed was never released, I wonder if it is worth mentioning?

Reply:
Good point, in fact I hesitated to include this line but see my next comment.

Reply:
I think we could say: the repo has been cleaned up and it now only contains released stuff (in particular odd roder & co were moved to other repos under the mathcomp organization)

Reply:
Ok, I'll do that.